### PR TITLE
fix(build): Generate umd and system files

### PR DESCRIPTION
**Pull request by romulo94**
Follow the thread: https://github.com/pmndrs/zustand/pull/740

Hi @dai-shi , I did the same config here.

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -87,18 +87,71 @@ function createCommonJSConfig(input, output) {
...
 }
 
+function createUMDConfig(input, output) {
+  return {
+    input,
+    output: {
+      file: output,
+      format: 'umd',
+      exports: 'named',
+      name: 'zustand',
```
Comment (dai-shi) [inline rollup.config.js:97]:
```suggestion
      name: 'valtio',
```

File: rollup.config.js [hunk 2]
```diff
@@ -87,18 +87,71 @@ function createCommonJSConfig(input, output) {
...
+      esModule: true,
+    },
+    external,
+    plugins: [
+      alias({
+        entries: {
+          './vanilla': 'valtio/vanilla',
+          '../vanilla': 'valtio/vanilla',
```
Comment (dai-shi) [inline rollup.config.js:105]: So, this is the trick.

Comment (romulo94) [reply rollup.config.js:104]: Yeah! I'm following the same config for alias

**Review by dai-shi — APPROVED** [review]
👍

**Pull request merged**